.github/workflows/hadolint.yml

```diff
@@ -0,0 +1,20 @@
+on:
+  workflow_call:
+jobs:
+  hadolint:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Checkout Code
+        uses: actions/checkout@v3
+      -
+        name: Install hadolint
+        run: | 
+          sudo wget -O /usr/bin/hadolint https://github.com/hadolint/hadolint/releases/download/v2.7.0/hadolint-Linux-x86_64
```

Comment:
Just curious why we are using v2.7.0, I saw the latest hadolint version is v2.12.0 over here https://github.com/hadolint/hadolint/releases?utm_source=thenewstack&utm_medium=website&utm_content=inline-mention&utm_campaign=platform .

Reply:
oh thanks for pointing it out. Will switch to the latest version